tests/common/devices.py

```diff
@@ -332,6 +332,36 @@ def critical_services_fully_started(self):
         logging.debug("Status of critical services: %s" % str(result))
         return all(result.values())
 
+    def get_critical_group_and_process_list(self, container_name):
```

Comment:
Function returns two lists, name should be plural to help indicate this: `get_critical_group_and_process_lists`

Reply:
Great suggestion! Updated.